variants/github_actions_ci/template.rb

```diff
@@ -1,3 +1,13 @@
 template "variants/github_actions_ci/workflows/ci.yml.tt", ".github/workflows/ci.yml"
 copy_file "variants/github_actions_ci/workflows/deploy_to_ec2.yml", ".github/workflows/deploy_to_ec2.yml"
 copy_file "variants/github_actions_ci/workflows/deploy_to_heroku.yml", ".github/workflows/deploy_to_heroku.yml"
+
+append_to_file "bin/ci-run" do
```

Comment:
IIRC the original goal of `bin/ci-run` was to let us easily mimic CI to an extent locally. 

Presumably this change means we can't run `bin/ci-run` locally without manually installing actionlint which breaks that. However I'm not sure how useful `bin/ci-run` is anymore given the matrix of stuff CI runs. One to discuss at guild I think.

Reply:
Yup that's what I was alluding to in my initial description.

That was the original purpose but it conflicts with using GitHub Actions effectively since it would require doing everything in monolithic jobs (among other things).

We've implicitly moved away from it years ago in our adoption of GHA but never actually discussed what to do with this script afaik - I think 2024 is a good year to have that discussion 😁

Reply:
Discussed in Ruby guild: no one has used `bin/ci-run` in years and think it's fine to subsume it for this purpose.

For now I'm going to leave it as is (i.e. not change the name or the fact that it gets included in the generated app), but we've talked about it for when we next have a good reason to do make actual changes to the file.